homie/support/repeating_timer.py

```diff
@@ -8,34 +8,25 @@
 
 
 class Repeating_Timer(object):
-    DEFAULT_INTERVAL_REDUCTION = 0.9
 
     """Repeat `function` every `interval` seconds."""
 
     def __init__(self, interval):
-        self.interval = int(interval * self.DEFAULT_INTERVAL_REDUCTION)
-
-        self.start = time.time()
+        self.interval = float(interval)
         self.event = Event()
-
         self.thread = Thread(target=self._target)
-        self.thread.setDaemon(True)
```

Comment:
why this change?

Reply:
hi, sorry this was me being possibly pedantic while going through this (to understand it myself). i was reviewing the documentation at https://docs.python.org/3/library/threading.html#thread-objects and it suggested in this case to use self.thread.daemon = True instead of setDaemon as a function. feel free to reject it would have no impact right now anyway